### ISSUE TITLE
New Pip 10 breaks deployments

### ISSUE THREAD

**Issue by Cediddi** [bobbyrenwick/ansible-pip#23]
Hi, 6 hours ago PyPA released pip10 and currently this role breaks the deployments. 

If I try to instal pip10 I get such error:
```
TASK [bobbyrenwick.pip : Upgrade to latest version of pip.] ********************
Saturday 14 April 2018  17:03:45 +0000 (0:00:00.029)       0:01:04.226 ******** 
fatal: [i_019e5b99c1f1fde37]: FAILED! => {"msg": "The conditional check 'pip_latest_output.stdout.find('Requirement already up-to-date') == -1' failed. The error was: error while evaluating conditional (pip_latest_output.stdout.find('Requirement already up-to-date') == -1): 'dict object' has no attribute 'stdout'"}
```

If I try to install pip9.0.3 I get another error:
```
TASK [bobbyrenwick.pip : Install required version of pip.] *********************
Saturday 14 April 2018  17:49:25 +0000 (0:00:00.163)       0:01:01.882 ******** 
fatal: [i_09c5d3ce100839b1b]: FAILED! => {"changed": false, "cmd": "pip install pip==9.0.3", "msg": "[Errno 2] No such file or directory", "rc": 2}
```

**Comment by nmusatti** [comment]
I managed to replicate these error messages on an installation that had "pip3", but no "pip" executable. Could this be your case?

**Comment by nmusatti** [comment]
Should be fixed by #33 . If this isn't the case, let us know.
